### PR TITLE
Nomenclature changes from Appdynamics to OpenTelemetry

### PR DESCRIPTION
**Pull request by DebajitDas**

### CODE REVIEW

File: instrumentation/otel-webserver-module/include/core/api/opentelemetry_ngx_api.h [hunk 1]
```diff
@@ -14,8 +14,8 @@
 * limitations under the License.
 */
 
-#ifndef APPDYNAMICS_NGX_API_H
-#define APPDYNAMICS_NGX_API_H
+#ifndef OPENTELEMETRY_NGX_API_H
```
Comment (kpratyus) [inline instrumentation/otel-webserver-module/include/core/api/opentelemetry_ngx_api.h:17]: Aren't these nomenclatures related to core directory?

File: instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_log.c [hunk 2]
```diff
@@ -31,7 +31,7 @@ void ngx_writeTrace(ngx_log_t *log, const char* funcName, const char* format, ..
 
     if (logState && log)
     {
-        ngx_log_error(NGX_LOG_ERR, log, 0, "mod_appdynamics: %s: %s", funcName, note);
```
Comment (kpratyus) [inline instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_log.c:34]: mod_'o'penetelemetry would be better?

**Review by kpratyus — CHANGES_REQUESTED** [review]

File: instrumentation/otel-webserver-module/include/core/api/opentelemetry_ngx_api.h (hunk 1 above)
Comment (DebajitDas) [reply instrumentation/otel-webserver-module/include/core/api/opentelemetry_ngx_api.h:17]: This nomenclature changes are done for all nginx code added. Gradually, its better we do the same for all other files in core directory.

File: instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_log.c (hunk 2 above)
Comment (DebajitDas) [reply instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_log.c:34]: ok will add here

**Review by kpratyus — APPROVED** [review]

**Pull request merged**